rotkehlchen/chain/evm/decoding/firebird_finance/decoder.py

```diff
@@ -0,0 +1,112 @@
+import logging
+from typing import TYPE_CHECKING, Any, Final
+
+from rotkehlchen.chain.ethereum.utils import asset_normalized_value
+from rotkehlchen.chain.evm.decoding.interfaces import DecoderInterface
+from rotkehlchen.chain.evm.decoding.structures import (
+    DEFAULT_DECODING_OUTPUT,
+    DecoderContext,
+    DecodingOutput,
+)
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import ChecksumEvmAddress
+from rotkehlchen.utils.misc import bytes_to_address
+
+from ..utils import maybe_reshuffle_events
```

Comment:
```suggestion
from rotkehlchen.chain.evm.decoding.utils import maybe_reshuffle_events
```
nitpick, but I don't think we generally use more than one dot in imports.

Reply:
thanks.